### DISCUSSION TITLE
Offline installer

### DISCUSSION THREAD

**Discussion by shobhit-deque** [theory/sqitch#805] [Q&A]
In our project, we are using sqitch, but for a customer who is not open to the internet, we need to download all dependencies and ship it in a zip or tar file. so I am looking for help to get sqitch available in download format and then on the client environment to be installed.

**Comment by theory** [comment]
Can they use Docker? If so you can [export the docker image and they can load it](https://stackoverflow.com/a/47946427/79202).

**Reply by shobhit-deque** [reply]
sorry, a non-docker version is required.

**Comment by shobhit-deque** [comment]
This is done, Thanks @theory

**Reply by theory** [reply]
Thanks for letting me know, I had been thinking of some ideas to suggest, and you freed me from that! Would you share how you solved this challenge?
